mkdocs_static_i18n/plugin.py

```diff
@@ -375,6 +376,12 @@ def on_files(self, files, config):
                     "search"
                 ]
 
+            if "with-pdf" in config["plugins"]:
+                output_path = self.i18n_configs[language]["plugins"]["with-pdf"].config["output_path"]
```

Comment:
In you [mkdocs-with-pdf PR](https://github.com/orzih/mkdocs-with-pdf/pull/68/files#diff-25f3fc84b0441a364fc5756f0891cc7aee14e4c3aeb0522d5b938a765c236f9dR56) you are proposing that the `output_path` option can be `None` which will cause a problem below

Reply:
Okay I am not quite sure about the None parameter here. I didn't changed that since I thought that the schema tell's the output_path has a default value:
https://github.com/orzih/mkdocs-with-pdf/pull/68/files#diff-25f3fc84b0441a364fc5756f0891cc7aee14e4c3aeb0522d5b938a765c236f9dL19

Reply:
```python
('output_path', config_options.Type(str, default="pdf/document.pdf"))
```

indeed, then if something **ensures it has a value**, please do not use a default set on get

Reply:
https://github.com/orzih/mkdocs-with-pdf/pull/68/commits/b0c7d97790f1575fc8ca11363914ef5fef4afd2c

Reply:
`self.output_path = config["plugins"]["with-pdf"].config["output_path"]` simply then?

Reply:
your right 😄 https://github.com/orzih/mkdocs-with-pdf/pull/68/commits/21b12956fc2d1d60e352c5c117e4cb01afa9a9d3